manifests/metering-config/example-shared-storage.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: Metering
+metadata:
+  name: "operator-metering"
+spec:
+  reporting-operator:
+
+    spec:
+     config:
+       defaultStorage:
+         create: true
+         name: "local-storage"
+         isDefault: true
+         type: "hive"
+         hive:
+           tableProperties:
+             location: "/user/hive/warehouse"
+
+  presto:
+    spec:
+      config:
+        sharedVolume:
+          enabled: true
+          storage:
+            persistentVolumeClaimStorageClass: REPLACEME
+      presto:
```

Comment:
None of the `image.tag` config options are necessary. Remove them.

Reply:
You need to remove the empty `presto:` and empty `hive:` here, and just below too, this is like setting it to null currently, which isn't correct.